src/wam2layers/config.py

```diff
@@ -112,7 +112,7 @@ class Config(BaseModel):
 
     Should be formatted as: `"YYYY-MM-DD[T]HH:MM"`. Start date < end date, even if
     backtracking.
-    When forward tracking the track_end_date is not written out as output date. 
+    When forward tracking the track_end_date is not written out as output date.
```

Comment:
Only a comment on forward tracking, should we say something on backward tracking here as wel?l